### PR TITLE
Update limitations to include geo-replication

### PR DESCRIPTION
**Pull request by Peter-Msft**

### CODE REVIEW

**Comment by PRMerger3** [comment]
@Peter-Msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ShannonLeavitt** [comment]
#label:"aq-pr-triaged"

**Comment by PRMerger13** [comment]
@WilliamDAssafMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@WilliamDAssafMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by WilliamDAssafMSFT** [comment]
Thanks @Peter-Msft, I've added a couple commits to update that file's ms.date and add this same caveat to another related article, for `sp_renamedb`.

#sign-off

File: docs/relational-databases/system-stored-procedures/sp-renamedb-transact-sql.md [hunk 1]
```diff
@@ -45,14 +43,18 @@ sp_renamedb [ @dbname = ] 'old_name' , [ @newname = ] 'new_name'
   
 ## Return Code Values  
  0 (success) or a nonzero number (failure)  
+ 
+## Remarks
+
+It is not possible to rename an Azure SQL Database configured in an [active geo-replication](/azure/azure-sql/database/active-geo-replication-overview) relationship.
```
Comment (ShannonLeavitt) [inline docs/relational-databases/system-stored-procedures/sp-renamedb-transact-sql.md:49]:
```suggestion
It is not possible to rename an Azure SQL database configured in an [active geo-replication](/azure/azure-sql/database/active-geo-replication-overview) relationship.

```
Lowercasing the term to reflect generic reference.

File: docs/relational-databases/databases/rename-a-database.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ monikerRange: "=azuresqldb-current||>=sql-server-2016||>=sql-server-linux-2017||
   - In SQL Server, you can set a database in single user mode to close any open connections. For more information, see [set the database to single-user mode](../../relational-databases/databases/set-a-database-to-single-user-mode.md).
   - In Azure SQL Database, you must make sure no other users have an open connection to the database to be renamed.
 - Renaming a database does not change the physical name of the database files on disk, or the logical names of the files. For more information, see [Database Files and Filegroups](database-files-and-filegroups.md#logical-and-physical-file-names).
+- It is not possible to rename an Azure SQL Database configured in an [active geo-replication](/azure/azure-sql/database/active-geo-replication-overview) relationship.
```
Comment (ShannonLeavitt) [inline docs/relational-databases/databases/rename-a-database.md:34]:
```suggestion
- It is not possible to rename an Azure SQL database configured in an [active geo-replication](/azure/azure-sql/database/active-geo-replication-overview) relationship.

```
Lowercasing the term to reflect generic reference.

**Comment by PRMerger7** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**